src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonPatchOperationsArrayProvider.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Microsoft.AspNetCore.Mvc.Formatters.Json
+{
+    public class JsonPatchOperationsArrayProvider : IApiDescriptionProvider
+    {
+        private readonly IModelMetadataProvider _modelMetadataProvider;
+
+        public JsonPatchOperationsArrayProvider(IModelMetadataProvider modelMetadataProvider)
+        {
+            _modelMetadataProvider = modelMetadataProvider;
+        }
+
+        /// <inheritdoc />
+        /// <remarks>
+        /// The order -999 ensures that this provider is executed right after the <c>Microsoft.AspNetCore.Mvc.ApiExplorer.DefaultApiDescriptionProvider</c>.
```

Comment:
Why doesn't `<see cref="DefaultApiDescriptionProvider"/>` work here? If it's just a missing `using`, add the needful.